tests/CollectionTest.php

```diff
@@ -259,6 +259,24 @@ public function testInsert(){
     $this->c->insert(-1, new TestClassA(5));
   }
 
+  public function testInsertEnd(){
+    $this->c->add(new TestClassA(1));
+    $this->c->add(new TestClassA(2));
```

Comment:
This was a pull request against Collection v1, which is not immutable, but Travis CI is running version two tests. Thanks for the input!